.github/workflows/lint.yml

```diff
@@ -5,79 +5,22 @@ on:
     branches: [main]
 
 jobs:
-  typecheck:
+  lint:
     if: ${{ !contains(github.head_ref, 'release/') }}
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4.1.0
-      - uses: taiga-family/ci/actions/setup-node@1.13.0
-      - run: npm run typecheck
-
-  cspell:
-    if: ${{ !contains(github.head_ref, 'release/') }}
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4.1.0
-      - uses: taiga-family/ci/actions/setup-node@1.13.0
-      - run: npm run cspell -- --no-progress
-
-  prettier:
-    if: ${{ !contains(github.head_ref, 'release/') }}
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4.1.0
       - uses: taiga-family/ci/actions/variables@1.13.0
-
       - uses: actions/checkout@v4.1.0
         if: env.SUPPORT_AUTO_PUSH == 'true'
         with:
           ref: ${{ github.head_ref }}
           token: ${{ secrets.TAIGA_FAMILY_BOT_PAT }}
-
       - uses: taiga-family/ci/actions/setup-node@1.13.0
-
+      - run: npm run typecheck
+      - run: npm run cspell -- --no-progress
```

Comment:
You've transformed parallel jobs into sequential jobs.

Is it actually required ? 🤔